backend/doorman/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM nginx

```

Comment:
Pls pin nginx to a version . This will fix the Codacy issue and get the build to pass